File: ELaHuo/ELaHuo/Classes/Setting/FAQViewController.m
//
//  FAQViewController.m
//  ELaHuo
//
//  Created by elahuo on 16/7/28.
//  Copyright © 2016年 elahuo. All rights reserved.
//

#import "FAQViewController.h"

@interface FAQViewController ()
@property (weak, nonatomic) IBOutlet UIWebView *FAQWebView;

@end

@implementation FAQViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.navigationItem.title = @"常见问题";
    
    NSString* path = [[NSBundle mainBundle] pathForResource:@"常见问题" ofType:@"html"];
    NSURL* url = [NSURL fileURLWithPath:path];
    NSURLRequest* request = [NSURLRequest requestWithURL:url] ;
    [self.FAQWebView loadRequest:request];
}


@end

File: eLahuoForCar/eLahuoForCar/view_controller/FAQViewController.swift
//
//  FAQViewController.swift
//  eLahuoForCar
//
//  Created by elahuo on 16/7/28.
//  Copyright © 2016年 elahuo.net. All rights reserved.
//

import UIKit

class FAQViewController: UIViewController {

    @IBOutlet weak var webView: UIWebView!
    
    
    override func viewDidLoad() {
        super.viewDidLoad()

        let path = NSBundle.mainBundle().pathForResource("常见问题", ofType: "html")
        let url = NSURL(fileURLWithPath: path!)
        let request = NSURLRequest(URL: url)
        webView.loadRequest(request)
    }

    
}
